.github/pull_request_template.md

```diff
@@ -26,5 +26,6 @@ dispatcher where…", "Improve our handling of…", etc.
 
 ## Tests and Linting
 - [ ] This branch is up-to-date with the `develop` branch.
+- [ ] The database diagram (`database/database_relationships.png`) has been updated if any database column have been changed, added, or dropped.
```

Comment:
This phrasing feels clunky. I'd certainly welcome any recommendations people may have.